Mathlib/Tactic/NormNum/Result.lean

```diff
@@ -194,7 +194,9 @@ A "raw rat cast" is an expression of the form:
 
 * `(Nat.rawCast lit : α)` where `lit` is a raw natural number literal
 * `(Int.rawCast (Int.negOfNat lit) : α)` where `lit` is a nonzero raw natural number literal
-* `(Rat.rawCast n d : α)` where `n` is a raw int cast, `d` is a raw nat cast, and `d` is not 1 or 0.
+* `(Rat.rawCast n d : α)` where `n` is a raw int literal, `d` is a raw nat literal, and `d` is not 1 or 0.
+
+(where a raw int literal is of the form `Int.ofNat lit` or `Int.negOfNat nzlit` where `lit` is a raw nat literal)
```

Comment:
Are you happy with my use of `nzlit` here?

Reply:
Sure, it's a bit awkward to phrase the constraint here (I think raw int literals are also defined in some other doc comment) but this is the right idea.